velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -247,7 +247,6 @@ class TpchBenchmark {
     } else {
       parquet::registerParquetReaderFactory(parquet::ParquetReaderType::DUCKDB);
     }
-    dwrf::registerDwrfReaderFactory();
```

Comment:
@majetideepak You also need to remove the registration of the Parquet reader factory above.

Reply:
You are right. I will fix this.

Reply:
Fixed in latest commit.